packages/pwa-buildpack/bin/buildpack

```diff
@@ -8,9 +9,8 @@ require('yargs')
     )
     .strict()
     .fail((msg, err) => {
-        if (!err.expected) {
-            throw err;
-        }
+        const error = err || msg;
+        prettyLogger.error(error.toString());
```

Comment:
It's a fallback error handler for an unrecognized command. If you comment it out and run `buildpack` with no arguments, you'll see the difference. It belongs in this ticket because it's about handling errors cleanly, and this fallback error handler will handle all errors in upcoming versions of `yargs`.